### DISCUSSION TITLE
population simulation ；5%-95%confidence interval

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1306] [Questions & Problems]
Hello everyone, I have been using PK-SIM modeling for my graduation project recently, and I found that the 5%-95% confidence interval of my group simulation was very narrow, and my senior students had a wider confidence interval than me before. What might be the reason for this?
![置信区间1](https://user-images.githubusercontent.com/95362728/190938103-c3b6eb01-5010-4172-85ec-67f3c213f75c.jpg)
![置信区间2](https://user-images.githubusercontent.com/95362728/190938110-d411b2d3-e590-462f-94d1-1cd5695d5f69.jpg)

**Comment by tobiasK2001** [comment]
Hi,

it is a bit hard to say why the concentration range of your model outputs are different, without knowing the used processes and population parameters.
One thing that is clear is that what you showed here from PK-sim is not a confidence interval. It is just the range of predicted concentrations based on the biometrics and number of individuals in your Population (including  interindividual variability). Your model parameters were, I guess, fitted to data using a model with a PK-Sim mean individual. In the Parameter Identification you will find the confidence interval of your estimated model parameters fitted to individual data, if you have done so and calculated sensitivity.
But when you propagate your model based on a mean individual to a Population, what you get from PK-Sim is more a "prediction interval" and this should not be confused with a Confidence interval you get e.g. from NLME models.

Hope that might help you.

Best, Tobias

**Reply by WTY-tt** [reply]
Thank you for your reply. It was very rewarding. I did not express the confidence interval accurately.
